src/Umbraco.Web/PropertyEditors/BlockEditorPropertyEditor.cs

```diff
@@ -118,7 +120,7 @@ public override object ToEditor(Property property, IDataTypeService dataTypeServ
                     return string.Empty;
                 }
 
-                if (blockEditorData == null || blockEditorData.BlockValue.ContentData.Count == 0)
+                if (blockEditorData == null)
```

Comment:
do you recall why this change was made?

Reply:
If you have a closer look to what exactly `blockEditorData = _blockEditorValues.DeserializeAndClean(val)` is doing (a few lines above), you will notice that we already do the check I removed ([`blockEditorData.BlockValue.ContentData.Count == 0`](https://github.com/umbraco/Umbraco-CMS/blob/d8b7e24b7f575e8b756f950e4de43a21e4da8795/src/Umbraco.Web/PropertyEditors/BlockEditorPropertyEditor.cs#L353)). And if the count is indeed 0, we will return `null` that would be caught in the single `is null` check 🙂

Reply:
Ahh yes, and `DeserializeAndClean` might return null, hence the null check 👍